posts/2020-09-01-jakarta-microprofile-beta-20.0.0.10.adoc

```diff
@@ -0,0 +1,336 @@
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in the Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. We've also made excellent TCK progress with these features.
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* jaspic-2.0
+* jacc-2.0
+* jpa-3.0 (The jpa-3.0 feature includes Eclipselink 3.0-RC1.)
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+* Jakarta XML Binding 3.0 (`jaxb-3.0`)
+* Jakarta Managed Bean 2.0 (`managedBeans-2.0`)
+* Concurrency 2.0 (`concurrent-2.0`)
+* Jakarta Enterprise Beans Home 4.0 (`ejbHome-4.0`)
+* Jakarta Enterprise Beans Lite 4.0 (`ejbLite-4.0`)
+* Bean Validation 3.0 (`beanValidation-3.0`)
+* CDI 2.0 (`cdi-2.0`)
+* WebSocket 2.0 (`websocket-2.0`; currently the integration with CDI is not completed)
+* JPA 3.0 (`jpa-3.0`)
+* JDBC 4.2 & 4.3 (`jdbc-4.2` & `jdbc-4.3`)
+* JTA 2.0 (`transaction-2.0`)
+* JSON-B 2.0 (`jsonb-2.0`)
+* JSON-P 2.0 (`jsonp-2.0`)
+* Servlet 5.0 (`servlet-5.0`)
+* JSP 3.0 (`jsp-3.0`)
+* Expression Language 4.0 (`el-4.0`)
+
+==== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 14, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty.beta</groupId>
+    <artifactId>openliberty-jakartaee9</artifactId>
+    <version>20.0.0.RELEASE_VERSION-beta</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.10-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want (but remember if you enable the `jsp-3.0` feature, the `servlet-5.0` and `el-4.0` features are automatically enabled for you) or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>jakartaee-9.0</feature>
+  </featureManager>
+----
+
+Or you can add the Web Profile convenience feature to enable all of the Jakarta EE 9 Web Profile beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>webProfile-9.0</feature>
+  </featureManager>
+----
+
+=== HEADING
+
```

Comment:
See if Kevin says there's any other news about the Jakarta EE 9 progress. If not, remove this section.

Reply:
Waiting for reply.

Reply:
Did you ask him in the channel or privately?

Reply:
Done